pom.xml

```diff
@@ -44,6 +51,16 @@
         <artifactId>RosettaCore</artifactId>
         <version>3.11.8</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
```

Comment:
can you set the `dep.guava.version` property instead?

Reply:
yes def, do you think I should just try to bump the guava version in the basepom, the issue here is needing `builderWithExpectedSize` which is added in 23

Reply:
Yeah, we can bump the version